res/css/_animations.pcss

```diff
@@ -77,6 +86,10 @@ limitations under the License.
         /* Override all keyframes in reduced-motion */
     }
 
+    @keyframes mx_spin {
+        /* Override all keyframes in reduced-motion */
```

Comment:
It is possible to enable zero transition here, but I am not quite sure if it should be enabled by default. 

Do users expect that the spin animation would be disabled by setting [user preferences](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion#user_preferences)?

If yes, I guess the animation should be disabled by default. Power users will find how to customize the animation anyway.

Reply:
According to https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion#user_preferences, the `reduce` request is honoured if:

- In GTK/GNOME: GNOME Tweaks > General tab (or Appearance, depending on version) > Animations is turned off.
- In Plasma/KDE: System Settings > Workspace Behavior -> General Behavior > "Animation speed" is set all the way to right to "Instant".
- In Windows 10: Settings > Ease of Access > Display > Show animations in Windows.
- In Windows 11: Settings > Accessibility > Visual Effects > Animation Effects
- In macOS: System Preferences > Accessibility > Display > Reduce motion.
- In iOS: Settings > Accessibility > Motion.
- In Android 9+: Settings > Accessibility > Remove animations.

Except macOS, the rest of the OSes tell users that animation is *removed* or *turned off*, rather than *reduced*.

Reply:
On this topic, see here: https://github.com/vector-im/element-web/issues/25475